components/_patterns/01-atoms/04-images/icons/icons.scss

```diff
@@ -0,0 +1,9 @@
+/// Mixin - With Icon
+/// Precede Text with SVG icon
+@mixin with-icon {
```

Comment:
Pretty sure we don't need this anymore. This was harder with SVG in the CSS background and doesn't apply to the new setup using the SVG element.

Reply:
Cool. I'll delete this file. I didn't know, so I didn't want to remove code that I wasn't familiar with.

Reply:
@evanmwillhite Do all of the other icon files stay there? e.g. Am I just deleting this scss file?

Reply:
yep!